parodos-model-api/src/main/java/com/redhat/parodos/workflow/task/BaseWorkFlowTask.java

```diff
@@ -116,6 +117,21 @@ public String getRequiredParameterValue(String parameterName) throws MissingPara
 				});
 	}
 
+	public boolean validateWorkflowParameters(WorkContext workContext) {
+		AtomicBoolean valid = new AtomicBoolean(true);
+		getWorkFlowTaskParameters().stream().filter(param -> !param.isOptional()).forEach(param -> {
+			try {
+				getRequiredParameterValue(param.getKey());
+			}
+			catch (MissingParameterException e) {
+				valid.set(false);
+				log.info("{} is missing from workContext for workflow task {}", param.getKey(), getName());
```

Comment:
pls change info level to warn or error.

Reply:
I'll set it to warn as it does not throw an exception directly